src/olympia/lib/crypto/tests/test_signing.py

```diff
@@ -364,7 +357,19 @@ def test_call_signing_promoted_recommended_only(self):
         recommendation_data = _get_recommendation_data(
             self.file_.current_file_path)
         assert recommendation_data['addon_id'] == 'xxxxx'
-        assert recommendation_data['states'] == ['recommended']
+        assert sorted(recommendation_data['states']) == states
```

Comment:
Could this use sets so order doesn't matter, or should `states` be sorted too?

Reply:
Casting the JSON into a set would lose the case where a state was included twice in the list, and we don't want that.  States could be sorted too, though it's a kwarg we pass in inside the tests so it's just a matter of remembering to define the kwarg value in alphabetical order or the test fails (we only pass a list of more than 1 item in one place at the moment).